### PR TITLE
Task/bws28 entity access management tables

### PR DESCRIPTION
**Pull request by ania-leichenko**

### CODE REVIEW

File: backend/src/common/enums/db/table-name.enum.ts [hunk 1]
```diff
@@ -1,6 +1,12 @@
 enum TableName {
   USERS = 'users',
   MASTERS = 'entity_access_management.masters',
+  PERMISSIONS = 'entity_access_management.permissions',
+  TENANTS = 'entity_access_management.tenants',
+  WORKERS = 'entity_access_management.workers',
+  GROUPS = 'entity_access_management.groups',
+  USERS_GROUPS = 'entity_access_management.users_groups',
+  GROUPS_PERMISSIONS = 'entity_access_management.groups_permission',
```
Comment (what1s1ove) [inline backend/src/common/enums/db/table-name.enum.ts]:
```suggestion
  GROUPS_PERMISSIONS = 'entity_access_management.groups_permissions',
```

File: backend/src/data/migrations/20220205143853_add_permissions_table.ts [hunk 2]
```diff
@@ -0,0 +1,20 @@
+import { Knex } from 'knex';
+import { TableName } from '~/common/enums/enums';
+
+const TABLE_NAME = TableName.PERMISSIONS;
+
+export async function up(knex: Knex): Promise<void> {
```
Comment (what1s1ove) [inline backend/src/data/migrations/20220205143853_add_permissions_table.ts]: export should be at the bottom (for all)

**Review by Yaroslaww-1 — DISMISSED** [review]

File: backend/src/data/migrations/20220206164137_edit_masters_table.ts [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { Knex } from 'knex';
+import { TableName } from '~/common/enums/enums';
+
+const TABLE_NAME = TableName.MASTERS;
+
+export async function up(knex: Knex): Promise<void> {
+  return knex.schema.table(TABLE_NAME, function (table) {
+    table.uuid('tenant_id').references('id').inTable(TableName.TENANTS);
```
Comment (what1s1ove) [inline backend/src/data/migrations/20220206164137_edit_masters_table.ts:8]: should be notNullable

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: backend/src/data/migrations/20220206164137_edit_masters_table.ts (hunk 3 above)
Comment (ania-leichenko) [reply backend/src/data/migrations/20220206164137_edit_masters_table.ts:8]: ok, but the scheme indicated --> https://prnt.sc/26pzdde

Comment (what1s1ove) [reply backend/src/data/migrations/20220206164137_edit_masters_table.ts:8]: got it. in this case, we don't need it 👍

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**